tests/kernel/smp/src/main.c

```diff
@@ -441,6 +444,105 @@ void test_wakeup_threads(void)
 	cleanup_resources();
 }
 
+/* a thread for testing SMP IPI */
+static void thread_get_cpu_entry(void *p1, void *p2, void *p3)
```

Comment:
You want to build and run this test only on systems where CONFIG_SCHED_IPI_SUPPORTED=n.  There are some systems which have SMP enabled but do not have a working interprocessor interrupt.

Reply:
Sorry, this is my error. I forgot modified the comment for this function. And this test_get_cpu() should be tested both with/without CONFIG_SCHED_IPI_SUPPORTED, I guess.